web/src/pages/Resolver/Parameters/Court.tsx

```diff
@@ -60,6 +60,7 @@ const Court: React.FC = () => {
           items={items}
           onSelect={(path: string | number) => typeof path === "string" && handleWrite(path.split("/").pop()!)}
           placeholder="Select Court"
+          value={`/courts/${disputeData.courtId}`}
```

Comment:
what would happen if `disputeData.courtId` is undefined? 🤔

Reply:
It would fallback to the `placeholder` value, which in our case is `Select a court`